cli/command/image/build.go

```diff
@@ -164,11 +161,6 @@ func NewBuildCommand(dockerCli command.Cli) *cobra.Command {
 	flags.SetAnnotation("squash", "experimental", nil)
 	flags.SetAnnotation("squash", "version", []string{"1.25"})
 
-	flags.BoolVar(&options.stream, "stream", false, "Stream attaches to server to negotiate build context")
```

Comment:
Should we add a hidden, deprecated dummy flag so that we can provide a useful error message if someone still uses this?

Similar to https://github.com/moby/moby/pull/24721/files

We can remove if after 1 or 2 releases

Reply:
This was experimental, so the best thing I can do is keep the flag to tell them it's been removed and that they should use buildkit, but I don't want to keep the feature.

Reply:
can we error out with human-friendly message when the dummy flag was specified?

Reply:
Sure, that's what I suggested in the comment above. I'll update the PR